### ISSUE TITLE
[Source Request]: Minneapolis MN USA

### ISSUE THREAD

**Issue by blevwal5866** [mampfes/hacs_waste_collection_schedule#2861]
### Municipality / Region

Minneapolis, MN

### Collection Calendar Webpage

https://apps.ci.minneapolis.mn.us/AddressPortalApp/Search?AppID=RecycleFinderApp

### Example Address

5866 Oakland Av S

### Collection Data Format

As html on a webpage

**Comment by jsnyder** [comment]
+1 for this. I've thought about adding it myself.

There's also the RSS XML format that the city provides:
http://apps.ci.minneapolis.mn.us/CalendarApp/Ex_CalendarRSS.aspx?linkurl=http://www.ci.minneapolis.mn.us/government/calendars.asp&datebook=Garbage%20and%20Recycling%20Thursday%20Route%20CD&type=rss

It is a bit weird that they didn't just go with a calendar format, but this shouldn't be too hard to parse as long as it doesn't change.

**Comment by jsnyder** [comment]
Looks like there's already an issue here: #2467

**Comment by 5ila5** [comment]
Added in #3240. Hope it keeps working as I get a CAPTCHA (but with a German IP)

This will be part of the next release or is available now as master (git) version 

install master/git: Developer tools -> actions -> select `update: Install update` -> click `Coose entity` and select WCS -> select `version` and type `master` in the input field -> run the action -> restart HA


<details><summary>Older versions of HACS:</summary>
<p>
install master with HACS version < 2.0: (3-dot menu -> redownload -> change version to master -> fully restart HA (this will increase update cycle by a lot)
</p>
</details>
